docs/package.json

```diff
@@ -11,7 +11,7 @@
     "flutter:copy": "cp -r ./flutter/authenticator/build/web public/flutter/authenticator",
     "build": "yarn build:props-tables && next build",
     "build:props-tables": "node --require esbuild-register ./scripts/generate-props-tables-data.ts",
-    "lint": "next lint",
+    "lint": "next lint && eslint . --ext ts,tsx,md,mdx",
```

Comment:
continue the conversation with @wlee221 https://github.com/aws-amplify/amplify-ui/pull/3369#discussion_r1092519411

I'm thinking of adding the lint at pre-commit time. What do you think?